ipatests/test_integration/test_otp.py

```diff
@@ -74,6 +77,32 @@ def kinit_otp(host, user, *, password, otp, success=True):
     )
 
 
+def ssh_2f(hostname, username, answers_dict, port=22):
+    """
+    :param hostname: hostname
+    :param username: username
+    :param answers_dict: dictionary of options with prompt_massage and value.
+    :param port: port for ssh
+    """
+    # Handler for server questions
+    def answer_handler(title, instructions, prompt_list):
+        resp = []
+        for prmpt in prompt_list:
+            prmpt_str = prmpt[0].strip()
+            resp.append(answers_dict[prmpt_str])
+        return resp
+    trans = paramiko.Transport((hostname, port))
+    trans.connect()
+    trans.auth_interactive_dumb(username, answer_handler)
+
+
+def set_sssd_conf(host, add_contents):
```

Comment:
I'm not entirely satisfied with this approach where you add a section to existing `sssd.conf` file. There is no code to remove the added section after the test, so it would affect other tests if they would be added here later.
Since SSSD is able to load configuration file fragments from `/etc/sssd/conf.d` directory, may be better would be to create a file in `/etc/sssd/conf.d/` and remove it afterwards?

Reply:
> I'm not entirely satisfied with this approach where you add a section to existing `sssd.conf` file. There is no code to remove the added section after the test, so it would affect other tests if they would be added here later.
@abbra Thanks for the review.
In finally block it is restored with original using 
`sssd_conf_backup.restore()`

Reply:
Ah, thanks, you are right. I think this should be OK.
Please remove the temp commit.